config/defaults.json

```diff
@@ -58,6 +58,10 @@
       "datapath": "/mnt/pelias/openaddresses",
       "files": []
     },
+    "polyline": {
+      "datapath": "/mnt/pelias/polyline",
```

Comment:
Add `"adminLookup": false,` here. It won't change anything obviously, but people outside Mapzen look at this file for a list of config options that can be changed.